buildSrc/src/main/java/Dependencies.kt

```diff
@@ -30,7 +30,7 @@ object Versions {
     const val disklrucache = "2.0.2"
     const val leakcanary = "2.4"
 
-    const val mozilla_appservices = "67.2.0"
+    const val mozilla_appservices = "68.0.0"
```

Comment:
I believe we need a new release of appservices that fixes the data-migration issue that we encountered during a previous attempt to update to v68. @travis79 can you think of any other outstanding nimbus functionality that we should try to get into such a release?

Reply:
And if there is nothing else urgent pending, would you like to try making a new nimbus-sdk release following the recently-added docs in the README, and I can incorporate it into an appservices release? 😁

Reply:
> I believe we need a new release of appservices that fixes the data-migration issue that we encountered during a previous attempt to update to v68. @travis79 can you think of any other outstanding nimbus functionality that we should try to get into such a release?

Right, this PR was what started all of that and I never updated the dependency. I don't know of anything that we _need_ to have in the update, but the on_telemetry_disabled one would be nice to have.

Reply:
Thanks! Version `v68.1.0` of appservices has been cut and is currently building, you can check whether it's ready by seeing if the little status dot on [this commit](https://github.com/mozilla/application-services/commit/1d57d7e5703ecaf09132b8161aaffe190eb2e1e6) has changed from yellow to green.